plugins/vendor/zoom/lib/model/oAuth/kZoomOauth.php

```diff
@@ -117,6 +118,19 @@ public static function verifyHeaderToken($zoomConfiguration)
 			}
 		}
 
+		if(isset($headers[self::X_ZM_SIGNATURE]) && isset($headers[self::X_ZM_REQUEST_TIMESTAMP]))
+		{
+			$signatureTimestamp = $headers[self::X_ZM_REQUEST_TIMESTAMP];
+			$secretToken = $zoomConfiguration[kOAuth::SECRET_TOKEN];
+			$body = file_get_contents(ZoomHelper::PHP_INPUT);
+			$signature = "v0=" . hash_hmac("sha256", "v0:$signatureTimestamp:$body", $secretToken);
+
+			if($signature == $headers[self::X_ZM_SIGNATURE])
+			{
+				return;
+			}
+		}
+
```

Comment:
log the signature we calculated compared to the one zoom sent

Reply:
done